docs/api/breaking-changes.md

```diff
@@ -81,6 +81,15 @@ powerMonitor.querySystemIdleTime(callback)
 const idleTime = getSystemIdleTime()
 ```
 
+## `app.enableMixedSandbox`
+
+```js
+// Deprecated
+app.enableMixedSandbox()
+```
+
+Mixed-sandbox mode is now enabled by default.
```

Comment:
Is there a way to disable it? Should it be mentioned here?

Reply:
it can be disabled by running Electron with `--no-sandbox`, @nornagon should we mention it? Also should this option be listed in `docs/api/chrome-command-line-switches.md`?

Reply:
We also have the environment variable added in https://github.com/electron/electron/pull/16576, which seems to be deliberately undocumented